src/main/java/org/example/Character/Mappers/MapFeatures.java

```diff
@@ -15,8 +15,10 @@ public List<Feature> mapClassFeatures(String JSON){
         try {
             JsonNode jsonNode = MAPPER.readTree(JSON);
             List <Feature> featureList = new ArrayList<>();
-            Feature feature = new Feature(jsonNode.get("results").findValuesAsText("name").toString());
-            featureList.add(feature);
+            Object[] feature = jsonNode.get("results").findValuesAsText("name").toArray();
```

Comment:
Does this return a full list or a long string?

Reply:
Return list of objects.
I can also change this:
featureList.add(new Feature(jsonNode.get("results").findValuesAsText("name").toString()));
result the same!